kubernetes/cache/cache.go

```diff
@@ -172,6 +179,11 @@ func NewKialiCache(namespaceSeedList ...string) (KialiCache, error) {
 
 	cacheNamespacesRegexps := make([]regexp.Regexp, len(cacheNamespaces))
 	for i, ns := range cacheNamespaces {
+		// '**' is a special character for selecting all namespaces
+		// but needs to be converted to an actual regexp.
+		if ns == "**" {
+			ns = ".*"
+		}
```

Comment:
Is this fixing some bug? I cannot figure out how it was working in the past.

Reply:
I added this awhile back and I honestly can't remember if it was a bug fix or not. I'll double check and remove if it works fine without it.